### ISSUE TITLE
Data readings need to be floats, not integers

### ISSUE THREAD

**Issue by ads04r** [ads04r/imouto-viewer#7]
Will require a DB migration, hence why it's in its own issue and I didn't just do it. Currently blocked by the progress of #6

**Comment by ads04r** [comment]
This doesn't need to be done any more, as I figured out an easier way of achieving the same result.
